Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesProvider.kt

```diff
@@ -33,16 +28,15 @@ class PersonCertificatesProvider @Inject constructor(
     }
 
     val personCertificates: Flow<Set<PersonCertificates>> = combine(
-        vaccinationRepository.vaccinationInfos,
-        testCertificateRepository.cwaCertificates,
-        recoveryCertificateRepository.cwaCertificates,
+        certificatesProvider.certificateContainer,
         personCertificatesSettings.currentCwaUser.flow,
         dccWalletInfoRepository.personWallets
-    ) { vaccPersons, tests, recoveries, cwaUser, personWallets ->
+    ) { certificateContainer, cwaUser, personWallets ->
+
+        val allCerts = certificateContainer.allCwaCertificates
+        val vaccPersons = certificateContainer.vaccinationInfos
```

Comment:
I see we still need to use the vaccination repo even though certificates have been provided by `certificateContainer` I prefer to keep it as before

Reply:
please,  isnt it cleaner to have a single source instead of importing every cert repo and compare their flows everytime?